content/operate/kubernetes/reference/yaml/rack-awareness.md

```diff
@@ -0,0 +1,120 @@
+
+Key permissions:
+- `nodes`: Read access to discover node zone labels
+- `get, list, watch`: Monitor node changes and zone assignments
+
+## Cluster role binding
+
+The [ClusterRoleBinding](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#rolebinding-and-clusterrolebinding) grants cluster-wide permissions to the service account.
+
+{{<embed-yaml "k8s/rack_aware_cluster_role_binding.md" "rack-aware-cluster-role-binding.yaml">}}
+
+Cluster role binding configuration:
+- `subjects.name`: Must match the service account name
+- `subjects.namespace`: Namespace where the operator is deployed
+- `roleRef.name`: Must match the cluster role name
+
+## Rack-aware Redis Enterprise cluster
+
+The rack-aware [REC configuration]({{< relref "/operate/kubernetes/reference/redis_enterprise_cluster_api" >}}) includes the `rackAwarenessNodeLabel` field.
+
+{{<embed-yaml "k8s/rack_aware_rec.md" "rack-aware-cluster.yaml">}}
+
+Rack-aware cluster configuration:
+- `metadata.name`: Cluster name (cannot be changed after creation)
+- `spec.rackAwarenessNodeLabel`: Node label used for zone identification
+- `spec.nodes`: Minimum 3 nodes, ideally distributed across zones
+
+Edit the values in the downloaded YAML file based on your environment, such as increasing nodes for better zone distribution, using custom zone labels, adding resource specifications, or enabling persistent storage.
+
+### Common zone labels
+
+Different Kubernetes distributions use different zone labels:
+- `Standard`: `topology.kubernetes.io/zone`
+- `Legacy`: `failure-domain.beta.kubernetes.io/zone`
+- `Custom`: Your organization's specific labeling scheme
+
+Verify the correct label on your nodes:
+
+```bash
+kubectl get nodes -o custom-columns=NAME:.metadata.name,ZONE:.metadata.labels.'topology\.kubernetes\.io/zone'
+```
+
+## Redis Enterprise database
+
+Database configuration for rack-aware clusters is the same as [basic deployments]({{< relref "/operate/kubernetes/reference/yaml-examples/basic-deployment#redis-enterprise-database" >}}).
+
+{{<embed-yaml "k8s/redb.md" "redis-enterprise-database.yaml">}}
+
+## Apply the configuration
+
+To deploy rack-aware Redis Enterprise clusters, follow [Deploy on Kubernetes]({{< relref "/operate/kubernetes/deployment" >}}) and ensure your Kubernetes nodes have proper zone labels. For detailed rack awareness configuration, see the [node selection recommendations]({{< relref "/operate/kubernetes/recommendations/node-selection" >}}).
+
+## Troubleshooting
+
+### Nodes not distributed across zones
+- Verify node labels are correct
+- Check that sufficient nodes exist in each zone
+- Ensure the `rackAwarenessNodeLabel` matches actual node labels
+
+### Cluster role permissions denied
+- Verify the ClusterRole and ClusterRoleBinding are applied
+- Check that the service account name matches in all resources
+
+### Database shards not distributed
+- Confirm the cluster has rack awareness enabled
+- Check that the database has multiple shards
```

Comment:
The rack awareness feature in the cluster distributes primary/replica across zones, but not necessarily different (primary) shards. Perhaps the instruction here should be to check that the database has replication enabled.